src/tools/auth0/handlers/brandingTemplates.js

```diff
@@ -0,0 +1,61 @@
+import DefaultHandler from './default';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    universalLogin: {
+      type: 'object',
+      properties: {
+        template: { type: 'string' }
+      }
+    }
+  }
+};
+
+export default class BrandingTemplatesHandler extends DefaultHandler {
+  constructor(options) {
+    super({
+      ...options,
+      type: 'brandingTemplates'
+    });
+  }
+
+  async getType() {
+    // in case client version does not support branding
+    if (!this.client.branding || typeof this.client.branding.getUniversalLoginTemplate !== 'function') {
+      return {};
+    }
+
+    // in case client version does not custom domains
+    if (!this.client.customDomains || typeof this.client.customDomains.getAll !== 'function') {
+      return {};
+    }
+
+    // if there's no custom domains, skip, feature requires custom domain.
```

Comment:
Because custom domains are required for this feature, I think it makes sense to bail out early if there's none configured.

Reply:
Is this requirement documented somewhere? I haven't personally tested new branding templates, so excuse my unfamiliarity.

Reply:
Yep, the API returns a 400 if there's no custom domain configured. There's some public doc here (see first paragraph): https://auth0.com/docs/customize/universal-login-pages/universal-login-page-templates

Reply:
Got it. Thanks!